cypress/integration/finance/funds/funds.deleteBudgetWithTransfer.spec.js

```diff
@@ -0,0 +1,37 @@
+import Funds from '../../../support/fragments/finance/funds/funds';
+import TopMenu from '../../../support/fragments/topMenu';
+import { getCurrentFiscalYearCode } from '../../../support/utils/dateTools';
+import testType from '../../../support/dictionary/testTypes';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import newFund from '../../../support/fragments/finance/funds/newFund';
+
+describe('ui-finance: Delete budget with transfer transaction from fund', () => {
+  const fundFrom = { ...newFund.defaultFund };
+  const fundTo = { ...newFund.defaultFund };
+
+  it('C343240 delete budget with transfer transaction', { tags: [testType.smoke] }, () => {
```

Comment:
see in testrail a lot of UI steps. I suggest to add TODO to redesign test in testrail to business actions.

Reply:
All UI steps reflected in script except verification for messages that disappear (previously we've decided that we won't verify it in automation scripts) - instead of these messages I'm checking budget values to prove that budget was not deleted or was created.

Reply:
Done